ethpm_cli/config.py

```diff
@@ -77,7 +90,15 @@ def get_w3(chain_id: int) -> Web3:
     infura_url = f"{SUPPORTED_CHAIN_IDS[chain_id]}.infura.io"
     headers = build_http_headers()
     infura_url = build_infura_url(infura_url)
-    return Web3(load_provider_from_uri(infura_url, headers))
+    w3 = Web3(load_provider_from_uri(infura_url, headers))
+
+    if private_key:
```

Comment:
*nitpick*

I'm in favor of more precise checks which in this case would be `if private_key is not None`